ush/nfcens/evs_wave_leadaverages.sh

```diff
@@ -23,7 +23,10 @@ ptype='lead_average'
 export GRID2OBS_CONF="${PARMevs}/metplus_config/${STEP}/${COMPONENT}/${RUN}_${VERIF_CASE}"
 
 cd ${DATA}
-mkdir ${DATA}/sfcshp
+if [ ! -s ${DATA}/sfcshp ]; then
+	mkdir ${DATA}/sfcshp
+fi
+
```

Comment:
Hi @SamiraArdani-NOAA. It looks like you are trying to stop getting the error you get in a log file when you try to make a directory that already exists. Instead of what you have added, I think that `mkdir -p ${DATA}/sfcshp` would accomplish the same thing and be a smaller change. NCO will be comparing EVS v1.0.0 to EVS v1.0.1 and asking for an explanation ever line that is changed. If this could be a smaller change (just adding this -p), that would be better. Thanks!

Reply:
@AliciaBentley-NOAA : The code was updated by using mkdir -p (adding only the -p) compare to original.